components/google_slides/README.md

```diff
@@ -11,3 +11,5 @@ Using the Google Slides API, you can build applications that do the following:
 - Crop images
 - Add or remove comments
 - And more!
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Reply:
Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).